client/src/components/ManageAccount/ManageAccount.js

```diff
@@ -68,15 +68,12 @@ const ManageAccountView = () => {
           <LoadingSpinner />
         ) : (
           <>
-            {errors.data && (
-              <SectionMessage type="error">
-                Unable to get your profile data
-              </SectionMessage>
-            )}
-            {errors.fields && (
-              <SectionMessage type="error">
-                Unable to get form fields
-              </SectionMessage>
+            {(errors.data || errors.fields) && (
+              <div className="manage-account-error">
+                <SectionMessage type="error">
+                  An error occurred loading your profile information.
```

Comment:
@edmondsgarrett, Reminder: Craig remembers y'all had a chat, and because this error is thrown if any data in this section fails to load, then the message should be changed to:

```suggestion
                  An error occurred loading your account information.
```

Yes? No?

Reply:
Yes, I remember this, and recall that the wording specified included 'profile'. If using 'account' would be better/more accurate, I can definitely change it.

Reply:
Okay. Yup, I saw the "profile" wording [in Slack](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1635187546000700?thread_ts=1634830617.001700&cid=CQUA4D5KJ). neither that nor this conditional wording got added to the ticket. Sorry.

Please, change to "account".

Reply:
Multiple private conversation are not easy to follow. I retract the change request until confirmed.

Reply:
Woo, okay. Confirmed, please use "account information" instead of "profile information".

Reply:
Moved to [reduced comment in new review](https://github.com/TACC/Core-Portal/pull/543#discussion_r747056306).